Code/Framework/AzToolsFramework/AzToolsFramework/API/ToolsApplicationAPI.h

```diff
@@ -605,6 +605,18 @@ namespace AzToolsFramework
          * system simulates its execution, and avoids some code duplication.
          */
         virtual void RunRedoSeparately(UndoSystem::URSequencePoint* redoCommand) = 0;
+
+        /**
+         * Create a prefab out of the entities provided, at the path provided.
+         * Automatically detects descendants of entities, and discerns between entities and child instances.
+         */
+        virtual bool CreatePrefab(
```

Comment:
I am more keen on not having magic bools as arguments, as it makes difficult to tell "what does this bool do".
For that I would recommend to have here to functions instead:

CreatePrefabInDisk()
CreatePrefabInMemory()

Reply:
Fixed.